docs/source/multidc/multidc.md

```diff
@@ -561,3 +561,41 @@ UN  172.16.39.209  336 KB     256          ?       7c30ea55-7a4f-4d93-86f7-c8817
 UN  172.16.25.18   759 KB     256          ?       665dde7e-e420-4db3-8c54-ca71efd39b2e  a
 UN  172.16.87.27   503 KB     256          ?       c19c89cb-e24c-4062-9df4-2aa90ab29a99  c
 ```
+
+## Scylla Manager
+
+To integrate a multi-datacenter ScyllaDB cluster with Scylla Manager, you must deploy the Scylla Manager in only one datacenter.
+
+In this example, let's choose Kubernetes Cluster deployed in first datacenter to host it.
+To deploy Scylla Manager, follow the steps described in [Deploying Scylla Manager on a Kubernetes Cluster](../manager.md)
+in ScyllaDB Operator documentation. 
+
+In order to define the Scylla Manager tasks, add them to the ScyllaCluster object deployed in the same Kubernetes cluster 
+in which your Scylla Manager is running.
+
+Each Scylla Operator instance creates a default Scylla Manager Agent auth token when ScyllaCluster is created, 
+because each Scylla Operator instance generates a unique token, you need to make sure that the entire cluster uses the same auth token.
```

Comment:
I suppose this happens even within a single kube cluster, right?
```suggestion
Every datacenter (represented by ScyllaCluster CR) is, by default, provisioned with a new, random Scylla Manager Agent auth token. To use Scylla Manager with multiple datacenter (represented by ScyllaClusters), you have to make sure they all use the same token.
```

Reply:
done